File: UnrealMatch/UnrealMatch/Resources/Scripts/Entities/Primitives/Size.ts
module Entities
{
    // Present a size on canvas for smth element        
    export class Size
    {
        public Width: number;
        public Height: number;

        constructor(w: number, h: number)
        {
            this.Width = w;
            this.Height = h;
        }

        public ReturnMultiplied(multiplier: number): Size
        {
            return new Size(this.Width * multiplier, this.Height * multiplier);
        }
    }
} 

File: UnrealMatch/UnrealMatch/Entities/Primitives/Size.cs
namespace UnrealMatch.Entities.Primitives
{
    public class Size
    {
        public int Width { get; set; }
        public int Height { get; set; }

        public Size(int w, int h)
        {
            this.Width = w;
            this.Height = h;
        }
    }
}